specification/common-types/resource-management/v3/types.json

```diff
@@ -653,15 +760,16 @@
       "type": "string",
       "minLength": 1,
       "maxLength": 90,
+      "pattern": "^[-\\w\\._\\(\\)]+$",
```

Comment:
Can we remove this pattern?

Reply:
@pshao25 Why, what problem does it cause?